signatures/example_signatures.json

```diff
@@ -0,0 +1,90 @@
+{
+    "ipSignature": [
+        "^176\\.191\\.129\\.\\d{1,3}$",
+        "^85\\.90\\.156\\.157$",
+        "^161\\.35\\.27\\.209$",
+        "^134\\.119\\.219\\.71$",
+        "^103\\.227\\.8\\.42$"
+    ],
+    "requestSignature": [
+        {
+            "method": "^GET$",
+            "uri": "^\/index.php$"
```

Comment:
I think you need to write "\\/index.php" for the match to work, since one \ gets removed I think when converting to JSON.
Have you tried to match against one of those signatures?

Reply:
I don't exactly get what you mean. The following code yields `true` for me:
```js
content = JSON.parse(fs.readFileSync("../signatures/example_signatures.json"));
console.log("/index.php".match(content.requestSignature[0].uri) !== null);
```
And if I try to write `"\/index.php"` my autoformatter removes the backslash. Is this what you mean?

Reply:
ok I think one backslash got removed in my comment. In the test yesterday, I had to use two backslashes before /index.php.
Otherwise web_ids wouldn't match against a log containing index.php.
But to generally avoid this slash/backslash escaping, we could just write "index.php", since we don't care at what position this string is in the request

Reply:
Ok, I have tried it now on the server, and it seems to work now with all variations '\\/index.php', '\/index.php' and '/index.php'
Looking at the regex documentation [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions), forward slashes are used to define start and end of a regex statement. I guess internally the characters are slashes are escaped correctly when executing the match.

In this case I woul remove backslashes that escape forward slashes, as you already mentioned.

Reply:
So do I need to remove them from the signature file aswell?

Reply:
Not sure. The request is still matched, if we keep the signature as is.
It is just about the readability and to see, that you don't need to escape a forward slash.

For later users, it is probably better to have examples with no backslash escaping.

Reply:
Ok, I will replace the escaped slashes with normal slashes